pkg/k8s/crdutils/register.go

```diff
@@ -71,20 +76,30 @@ func NewCRD(crdName, resName string, crd apiextensionsv1.CustomResourceDefinitio
 func RegisterCRDs(clientset apiextensionsclient.Interface, crds []CRD) error {
 	g, _ := errgroup.WithContext(context.Background())
 	g.Go(func() error {
-		return createCRDs(clientset, crds)
+		return createCRDs(clientset, crds, CRDOptions{})
+	})
+
+	return g.Wait()
```

Comment:
nit: how about just `return RegisterCRDsWithOptions( CRDOptions{})` here?

Reply:
It makes sense, thank you! I've replaced duplicated code